task/rpms-signature-scan/0.1/rpms-signature-scan.yaml

```diff
@@ -5,19 +5,27 @@ metadata:
   name: rpms-signature-scan
 spec:
   params:
-    - name: INPUT
+    - name: image-url
```

Comment:
This is a breaking change. Do we need to create a new version of the task or all users of the task are aware of this?

Reply:
Its a new task that we just created few days ago, we are modifying everything before adding it to the template and before its been used by anyone.